appconfig/settings/tradeFeed.q

```diff
@@ -1,4 +1,4 @@
 
 \d .servers
 
-CONNECTIONS:`gateway`rdb         // if connectonstart false, include tickerplant in tickerplanttypes, not in CONNECTIONS
+CONNECTIONS:enlist`segmentedtickerplant         // if connectonstart false, include tickerplant in tickerplanttypes, not in CONNECTIONS
```

Comment:
Will removing the gateway have any impact on the behavior of the tradeFeed? I'm wondering if there is a reason it was included in the first place

Reply:
No impact in removing gateway. Did a full round of testing for it and all good. No idea why gateway was there to begin with.